src/object.d

```diff
@@ -3257,8 +3252,6 @@ private size_t getArrayHash(in TypeInfo element, in void* ptr, in size_t count)
 }
 
 
-// @@@BUG5835@@@ tests:
-
```

Comment:
I didn't see a point to it.

https://issues.dlang.org/show_bug.cgi?id=5835

After fixes have been pulled, it takes a strong reason to keep references to the bugzilla entry in the source code. (Otherwise things would fill up with clutter.)